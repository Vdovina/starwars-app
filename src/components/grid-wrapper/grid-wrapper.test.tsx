import { create, act as testAct, ReactTestRenderer } from 'react-test-renderer';
import { GridWrapper } from './';

describe('GridWrapper Tests', () => {
  it('GridWrapper render', () => {
    let component: ReactTestRenderer;
    testAct(() => {
      component = create(
        <GridWrapper>
          <div>Block1</div>
          <div>Block2</div>
          <div>Block3</div>
          <div>Block4</div>
        </GridWrapper>
      );
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
});
