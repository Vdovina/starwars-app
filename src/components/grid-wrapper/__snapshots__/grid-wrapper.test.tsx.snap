// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridWrapper Tests render 1`] = `
<div
  className="item-card"
>
  <div>
    Block1
  </div>
  <div>
    Block2
  </div>
  <div>
    Block3
  </div>
  <div>
    Block4
  </div>
</div>
`;
